datafusion/src/physical_plan/expressions/try_cast.rs

```diff
@@ -236,7 +236,7 @@ mod tests {
     #[test]
     fn invalid_cast() {
         // Ensure a useful error happens at plan time if invalid casts are used
-        let schema = Schema::new(vec![Field::new("a", DataType::Int32, false)]);
+        let schema = Schema::new(vec![Field::new("a", DataType::Null, false)]);
```

Comment:
Int32 -> LargeBinary cast is valid in arrow2, therefore I change to another unacceptable case `Null` here